source/nodejs/adaptivecards/example.html

```diff
@@ -53,7 +53,6 @@
 			adaptiveCard.hostConfig = new AdaptiveCards.HostConfig({
 				fontFamily: "Segoe UI, Helvetica Neue, sans-serif"
 				// More host config options
-				// You can find example configs here: https://github.com/microsoft/AdaptiveCards/tree/master/samples/HostConfig 
```

Comment:
curious -- why?

Reply:
Sorry, why what?

Reply:
Guessing you're asking why the more informative line was removed - I didn't remove it. It's most likely due to automatic conflict resolution.

Reply:
yup - got it - then let's revert this file before the merge; 

---
In reply to: [297811427](https://github.com/microsoft/AdaptiveCards/pull/2995#discussion_r297811427) [](ancestors = 297811427)

Reply:
I re-added that comment.